src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/IntToVisibilityConverter.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class IntToVisibilityConverter : IValueConverter
```

Comment:
This looks almost identical to the DownloadCountToVisibilityConverter. Can we repurpose that, perhaps rename it to something more generalized, and use it for number to visibility conversion?

Reply:
> I'm having a hard time trying to figure out how we could repurpose the DownloadConverter, usually something grater than 0 is true, else is false, but this is considering 0 as true.

I don't follow your comment here as neither converter returns booleans, but they are slightly different.

- DownloadCount returns visible for >=0.
- You're wanting a positive non-zero value (>0), and I agree that's different.

You could create one that takes options to determine if 0 is considered visible or not.

I also think it's ok as is, but recommend renaming this new one to be more specific to the property or, maybe: `PositiveIntToVisibilityConverter` ?

Consider instead of binding XAML to integers directly to instead create a ViewModel property that's a boolean, then you can just use the `BoolToVisiblityConverter`.